registration/api/v2/views.py

```diff
@@ -235,3 +238,92 @@ def get_queryset(self):
             return BetaTestApplication.objects.filter()
         else:
             return BetaTestApplication.objects.filter(user=self.request.user)
+
+
+class OpenEdxInstanceDeploymentViewSet(RetrieveDestroyAPIView, GenericViewSet):
```

Comment:
Shouldn't this endpoint should also have a create option? It could take over the current commit changes API.

Reply:
The frontend already needs to wait to perform a query on this, it waits until the `selectedInstance` is available to make the query, and only pulls the required data.

I prefer to avoid a list endpoint because it'll result in a huge number of queries due to get `instance` -> `appserver` for each betatest application and it's not really required.